### ISSUE TITLE
can you provide objects365 pretrain model 

### ISSUE THREAD

**Issue by lixinye666** [Deci-AI/super-gradients#1740]
### 💡 Your Question

hello , any guys train objects365 model ?

**Comment by BloodAxe** [comment]
O365 pretraining was part of training process of YoloNAS but it is not going to be released as getting perfect results on this dataset was never an end goal. You are more than welcome to fine-tune YoloNAS on O365 by yourself.
